packages/ui/src/composites/wui-list-button/index.ts

```diff
@@ -0,0 +1,31 @@
+import { html, LitElement } from 'lit'
+import { property } from 'lit/decorators.js'
+import '../../components/wui-text/index.js'
+import { elementStyles, resetStyles } from '../../utils/ThemeUtil.js'
+import { customElement } from '../../utils/WebComponentsUtil.js'
+import styles from './styles.js'
+
+@customElement('wui-list-button')
+export class WuiListButton extends LitElement {
+  public static override styles = [resetStyles, elementStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+  @property() public text = ''
+
+  @property({ type: Boolean }) public disabled = false
+
+  // -- Render -------------------------------------------- //
+  public override render() {
+    return html`
+      <button ?disabled=${this.disabled} ontouchstart>
```

Comment:
Why do we have `ontouchstart`? I saw this on Tooltip component as well? What it helps for?

Reply:
To enable touch animations on `mobile` devices